packages/wdio-utils/src/shim.ts

```diff
@@ -152,6 +152,28 @@ let wrapCommand = function wrapCommand<T>(commandName: string, fn: Function, pro
             Promise.resolve(promise).then((ctx: Clients.Browser) => cmd.call(ctx, ...args)),
             {
                 get: (target, prop: string) => {
+                    /**
+                     * handle symbols, e.g. async iterators
+                     */
+                    if (typeof prop === 'symbol') {
+                        return () => ({
+                            i: 0,
+                            target,
+                            async next () {
+                                const elems = await this.target
+                                if (!Array.isArray(elems)) {
+                                    throw new Error('Can not iterate over non array')
+                                }
+
+                                if (this.i < elems.length) {
+                                    return { value: elems[this.i++], done: false }
```

Comment:
Shouldn't this be like below?
```suggestion
                                    return Promise.resolve({ value: elems[this.i++], done: false })
```